migrations/20221217114046-create-social-links-table.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+import { SocialLinkType } from '../server/models/SocialLink';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('SocialLinks', {
+      CollectiveId: {
+        type: Sequelize.INTEGER,
+        references: { model: 'Collectives', key: 'id' },
+        onDelete: 'CASCADE',
+        onUpdate: 'CASCADE',
+        allowNull: false,
+        primaryKey: true,
+      },
+      type: {
+        type: Sequelize.STRING,
```

Comment:
:ice_cream: I would have preferred an enum there, but it's personal; enums require more work to support new values or when refactoring.